tests/planner.lua

```diff
@@ -0,0 +1,87 @@
+local TestEZ = script.Parent.Parent.TestEZ
+local TestPlanner = require(TestEZ.TestPlanner)
+local TestBootstrap = require(TestEZ.TestBootstrap)
+local TestEnum = require(TestEZ.TestEnum)
+
+local testRoot = script.Parent.planning
+
+local function verifyPlan(plan, expected, notSkip)
+	local nodes = plan:findNodes(function(node)
+		return (not notSkip) or (node.modifier ~= TestEnum.NodeModifier.Skip)
+	end)
+
+	local nodeNames = {}
+	for _, node in pairs(nodes) do
+		table.insert(nodeNames, node:getFullName())
+	end
+
+	local pass = true
+	local message = ""
+
+	for _, e in pairs(expected) do
+		local ok = false
+		for _, g in pairs(nodeNames) do
```

Comment:
What do `e` and `g` mean here (expected/got?), and can we expand their names?

If these are lists, we should also iterate them with `ipairs` to help indicate that.

Reply:
Will do, but expected is already used, so I'll still have to abbreviate that or something. (Is `exp` clear enough?)